package.json

```diff
@@ -26,9 +26,9 @@
         "^1.0.0": "consumeToolbar"
       }
     },
-    "linter-plus-self": {
+    "linter-indie": {
```

Comment:
Depends on whether whatever is consuming the `required-packages` key supports version numbers as that service is only available with `linter` v2.